digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala

```diff
@@ -514,9 +514,11 @@ class VVHClient(vvhRestApiEndPoint: String) {
       "MTKHEREFLIP",
       "VALIDFROM",
       "GEOMETRY_EDITED_DATE",
+      "CREATED_DATE",
+      "LAST_EDITED_DATE",
       "SURFACETYPE").contains(x)
-    }.filter { case (_, value) =>
-      value != null
+    }.filter { case (key, value) =>
+      value != null || key == "LAST_EDITED_DATE" || key == "GEOMETRY_EDITED_DATE"
```

Comment:
to let the properties "LAST_EDITED_DATE" and "GEOMETRY_EDITED_DATE" go to extractedProperties, to be treated after in RoadAddressService